tests/tests_ansible.yml

```diff
@@ -374,14 +374,162 @@
         register: result
         failed_when: result is failed or result is changed
 
+      - name: add multiple custom services
+        firewall_lib:
+          service: custom, another_custom
```

Comment:
This will make a service with the name "custom, another_custom" - if you want to pass in a list, you'll need to use a yaml list like
```yaml
  service: [custom, another_custom]
```
or
```yaml
  service:
    - custom
    - another_custom
```
The first method is usually preferred for very short lists like this